articles/healthcare-apis/dicom/deploy-dicom-services-in-azure-data-lake.md

```diff
@@ -206,11 +206,24 @@ Use the Azure portal to **Deploy a custom template** and then use the sample ARM
 
 1. When prompted, select the values for the workspace name, DICOM service name, region, storage account name, storage account SKU, and container name.  
 
-1. Select **Review + create** to deploy the DICOM service.  
+1. Select **Review + create** to deploy the DICOM service.
 
-## Next steps
+## Troubleshooting
+
+### Connectivity
 
+To be alerted to store health and connectivity failures, please sign up for [Resource Health alerts](https://learn.microsoft.com/en-us/azure/service-health/resource-health-alert-monitor-guide).
+
+### 424 Failed Dependency
+
+When the response status code is `424 Failed Dependency`, the issue lies with a dependency configured with DICOM and if may be the DataLake store.
```

Comment:
Typo, and I think data lake should be lowercase.

```suggestion
When the response status code is `424 Failed Dependency`, the issue lies with a dependency configured with DICOM and it may be the data lake store.
```